File: packages/gunreip/laravel-translation-workbench/resources/views/pages/tw-graph/samples/documentation/idea-to-paper/flow/while/code-examples/while-action-sequence.cs
var groups = LoadGroups(); // List<List<Item>>
var groupIndex = 0;
while (groupIndex < groups.Count) {
    var items = groups[groupIndex];
    PrepareGroup(items);
    var itemIndex = 0;
    while (itemIndex < items.Count) {
        ProcessItem(items[itemIndex]);
        itemIndex = itemIndex + 1;
    }
    FinalizeGroup(items);
    groupIndex = groupIndex + 1;
}
ShowSummary();

File: packages/gunreip/laravel-translation-workbench/resources/views/pages/tw-graph/samples/documentation/idea-to-paper/flow/while/code-examples/while-action-sequence.java
List<List<Item>> groups = loadGroups();
int groupIndex = 0;
while (groupIndex < groups.size()) {
    List<Item> items = groups.get(groupIndex);
    prepareGroup(items);
    int itemIndex = 0;
    while (itemIndex < items.size()) {
        processItem(items.get(itemIndex));
        itemIndex = itemIndex + 1;
    }
    finalizeGroup(items);
    groupIndex = groupIndex + 1;
}
showSummary();
